sklearn/metrics/_plot/tests/test_plot_confusion_matrix.py

```diff
@@ -264,3 +263,35 @@ def test_confusion_matrix_text_format(pyplot, data, y_pred, n_classes,
     text_text = np.array([
         t.get_text() for t in disp.text_.ravel()])
     assert_array_equal(expected_text, text_text)
+
+
+@pytest.mark.parametrize("values_format", [None])
+def test_confusion_matrix_standard_format(pyplot, data, y_pred, n_classes,
+                                          fitted_clf, values_format):
+    X, y = data
+    cm = np.array([[10000000, 0], [29, 123123]])
+    disp = plot_confusion_matrix(fitted_clf, X, y,
+                                 include_values=True,
+                                 values_format=values_format)
+    disp.confusion_matrix = cm
+    plotted_text_values = disp.plot().text_
+    # Values should be shown as whole numbers 'd',
+    # except the first number which should be shown as 1e+07
+    test = []
+    for i in plotted_text_values.ravel():
+        test.append(i.get_text())
+    assert test == ['1e+07', '0', '29', '123123']
```

Comment:
if these expected values are hardcoded in the test, it no loner makes sense for values_format to be a parameter. Alternatively, if you want to make values_format a parameter, the expected values must also be parameters.

Reply:
Sorry, you're totally right about that, seems silly to parametrize this indeed.